woocommerce/woocommerce-gateway-stripe
# Remove unused code related to old Stripe settings

Fixes #2431

## Changes proposed in this Pull Request:

Removes unused code that should've been removed in #2166 from the following files:

- `includes/class-wc-gateway-stripe.php`
- `includes/connect/class-wc-stripe-connect.php`
- `includes/payment-methods/class-wc-stripe-upe-payment-gateway.php`

Also, removes the minified file `assets/js/stripe-admin.min.js` given that the source file `assets/js/stripe-admin.js` was already removed in #2166








## Testing instructions

There's no way to test this. The removed code are some leftovers from the [original PR](https://github.com/woocommerce/woocommerce-gateway-stripe/pull/2166) where we wanted to remove the deprecated Stripe settings.

Just in case, we should try setting up the Stripe keys the usual way, to make sure it works.

- Go to WooCommerce > Settings > Payments
- Click "Manage" on the "Stripe" row.
- Click the "Settings" tab.
- Click "Edit account keys" in the "General" section.
- On the "Test" tab enter the "Test publishable key" and the "Test secret key"
- Click "Test connection". Connection should be successfull.
- Click "Save test keys". Keys should be persisted without issues.





---


**Post merge**
